src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -182,6 +182,57 @@ public void GarbageCollect(Func<string, IWorkloadResolver> getResolverForWorkloa
             }
         }
 
+        public string InstallWorkloadSet(CliTransaction transaction, string path)
```

Comment:
For all 'path' references, make it clear (pathToExtractedWorkloadSet) and include comments if possible